workshops/geoserver/intro/doc/source/install/geoserver.rst

```diff
@@ -0,0 +1,213 @@
+.. install.geoserver:
+
+OSGeo Community GeoServer Distribution

```

Comment:
Not sure about this title. The capital letters implies that this is a product, but this is the first time I've seen it referred to this way. Suggest calling it "GeoServer" or "GeoServer community distribution" (no caps)?\.